java-checks/src/test/files/checks/EqualsParametersMarkedNonNullCheck.java

```diff
@@ -0,0 +1,21 @@
+class test{
```

Comment:
you can remove the parent class `test`, it is not required. Just put all the classes at the same level.